src/Lib/2D/Surface.cpp

```diff
@@ -1629,7 +1630,7 @@ void Surface::bltStringInBox(const iRect &rect, const char *string, PIX color,
 
     // Remove any spaces.
     while (string[length] == ' ') {
-      pos.x += FONT_WIDTH;  // TODO REMOVE
+      pos.x += MenuConfig::menu_font_size;  // TODO REMOVE
```

Comment:
This is wrong - I don't think Surface has anything specific to do with menu fonts.

Reply:
a nice solution could be to support passing the font size in here. eventually all font rendering in Surface will take font sizes I imagine.

Reply:
I couldn't figure out what this value did. When I changed it, nothing appeared to happen. (Since it had the same value as font_size and had the name font in it, I assumed it was related to the font.) I can hardcode it back to 14, though.

Reply:
Yeah it has to do with fonts, but it's not specific to the menu AFAIK. Many other things in the game are using these text rendering methods, so if changing "menu font size" in Lua changes those too, that that's confusing, so it we should leave it hard coded for now (acceptable to fix separately) or pass in the size (best solution). A nice stop gap solution might be to copy pasta the method and update it to take a font size, and then gradually migrate the code base.